commit b4ab2fea7c797b0b8b92332c7e315703c12d37d8
Author: Chuck Lever <chuck.lever@oracle.com>
Date:   Fri Jul 30 16:07:36 2021 -0400

    SUNRPC: Add RPC_AUTH_TLS protocol numbers
    
    Shared by client and server. See:
    
    https://www.iana.org/assignments/rpc-authentication-numbers/rpc-authentication-numbers.xhtml
    
    Signed-off-by: Chuck Lever <chuck.lever@oracle.com>

diff --git a/include/linux/sunrpc/msg_prot.h b/include/linux/sunrpc/msg_prot.h
index 938c2bf29db8..02117ed0fa2e 100644
--- a/include/linux/sunrpc/msg_prot.h
+++ b/include/linux/sunrpc/msg_prot.h
@@ -20,6 +20,7 @@ enum rpc_auth_flavors {
 	RPC_AUTH_DES   = 3,
 	RPC_AUTH_KRB   = 4,
 	RPC_AUTH_GSS   = 6,
+	RPC_AUTH_TLS   = 7,
 	RPC_AUTH_MAXFLAVOR = 8,
 	/* pseudoflavors: */
 	RPC_AUTH_GSS_KRB5  = 390003,